app/jobs/create_release_notifications.rb

```diff
@@ -0,0 +1,18 @@
+class CreateReleaseNotifications < ApplicationJob
```

Comment:
We could join the job with the RSS feed checker I guess, it would mak sense to me.